src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -23,10 +23,10 @@ import useLocalize from '../../hooks/useLocalize';
 import * as ReceiptUtils from '../../libs/ReceiptUtils';
 import useWindowDimensions from '../../hooks/useWindowDimensions';
 import transactionPropTypes from '../transactionPropTypes';
-import Image from '../Image';
```

Comment:
@aswin-s 

This also uses react native Image internally. You can just pass the params and it should work. Not sure why we need to change the component?

Reply:
@abdulrahuman5196 Nope. For native components we are importing Image component from 'react-native-fast-image'

https://github.com/Expensify/App/blob/9398754f3add40d84c9c84c23b79f3abb6165fa2/src/components/Image/index.native.js#L2

Reply:
If we pass resize mode as 'repeat' for Image component from FastImage, it throws error on native platforms

![Screenshot_1693328280](https://github.com/Expensify/App/assets/6880914/bcde5f19-e900-4b1a-8bf3-2e299fd92a12)

Reply:
@aswin-s 
Oh got it. For native components we are using fast image. But still I wouldn't want us to change the components to fix this particular issue. 

We should explore on how to send proper values based on platform.(Not the previous code change to check platforms from array). Maybe we can do some tweaking in index.native.js

Reply:
But both the other places are using Image component from react-native to show the background image. So it is not a hack just to fix this issue. And since we are embedding the background image, none of the advantages of using fast-image (like caching) are useful in this case. Also note that [request for support](https://github.com/DylanVann/react-native-fast-image/issues/253) of repeat mode in fast image is open since 2018.

Reply:
@grgia Could you kindly confirm on this? We have been waiting on this for sometime?

Reply:
@abdulrahuman5196 @aswin-s 

I'd agree with @aswin-s 
> The only benefit of using fast-image library is improving load performance of remote images by caching. Which is not the case here. Background image is embedded with the binary

I think we can use the native library for this case